index.js

```diff
@@ -191,12 +191,29 @@ FileList.prototype = new (function () {
    * pattern for finding file to include in the list. Arguments should be strings
    * for either a glob-pattern or a specific file-name, or an array of them
    */
-  this.include = function (items) {
-    if (items) {
-      this.pendingAdd = this.pendingAdd.concat(items).filter(function (item) {
-        return !!item;
-      });
+  this.include = function () {
+    var args = arguments
```

Comment:
I might have upgraded `arguments` to a full array, using `var args = Array.prototype.slice.call(arguments);` so you can simply `map` over the items, but the ordinary for-loop works too.